MovieChecker/MovieChecker/MovieList.h

```diff
@@ -0,0 +1,24 @@
+//
+//  MovieList.h
+//  MovieChecker
+//
+//  Created by Joseph Goldberg on 1/27/15.
+//  Copyright (c) 2015 Vokal. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@protocol MovieListDelegate <NSObject>
+
+- (void)synchMoviesWithArray:(NSArray *)movies;
+
+@end
+
+@interface MovieList : NSObject
```

Comment:
`MovieList` reads like a model, but this is more like a controller... perhaps `MovieListController` or `MovieListLoader`?